cfn-signal/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+eval `ec2cluster`;
```

Comment:
I think you're missing something.  ec2cluster outputs a bunch of data like:

```
REGION=us-west-1
TAG_AWS_CLOUDFORMATION_STACK_ID=something_or_other
```

We need to put those variables into our environment by calling `eval`.  If I removed `eval`, calling ec2cluster would be absolutely pointless.